### ISSUE TITLE
Training Time with Batch Size = 1

### ISSUE THREAD

**Issue by Lionel-LS** [junyanz/pytorch-CycleGAN-and-pix2pix#1549]
HI!
I would like to know how long did it take for you to train the CycleGAN network for Horse/Zebra dataset with a Batch Size =1 on a high-end GPU?
Both for the first 100 epoch with constant learning rate and last 100 epoch with linearly decay (as for the training detailed in your publication).

I suppose it would last a very long time (weeks) on a single GPU!

Could you precise the GPU used and the memory size.

Thanks.

**Comment by junyanz** [comment]
It should take 1-2 days on a GTX 1080 GPU. I expect the training time to be much shorter as the GPUs have been much faster compared to 2017.
